upgrade_test.py

```diff
@@ -523,8 +527,8 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
             self.db_cluster.node_to_upgrade.check_node_health()
             self.fill_and_verify_db_data('after upgraded %s' % self.db_cluster.node_to_upgrade.name)
 
-        # wait for the 20m read workload to finish
-        self.verify_stress_thread(read_20m_cs_thread_pool)
+        # wait for the 80m read workload to finish
+        self.verify_stress_thread(read_80m_cs_thread_pool)
```

Comment:
again why?

Reply:
I explained in the commitlog: https://github.com/scylladb/scylla-cluster-tests/pull/1438/commits/0c89a8db8a4cc6804f6a96f0954e2af31d897de6